Issue with 'expo-yarn-workspaces' on windows

Hey there,

I tried installing on windows and got this:

error C:\Users\Master\Documents\Development\ZART-test\node_modules\@expo-next-monorepo-example\expo: Command failed.
Exit code: 1
Command: expo-yarn-workspaces postinstall && cd ../..
Arguments:
Directory: C:\Users\Master\Documents\Development\ZART-test\node_modules\@expo-next-monorepo-example\expo
Output:
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module 'C:\Users\Master\Documents\Development\ZART-test\node_modules\node_modules\expo-yarn-workspaces\bin\expo-yarn-workspaces.js'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Seems to pertain to this:
https://github.com/expo/expo/issues/6983

-------------------------------------------------------------
The fix for me was updating expo-yarn-workspaces from ^1.5.1 to version ^1.7.0

Cheers!

--- a/package.json
+++ b/package.json
@@ -38,7 +38,7 @@
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-react": "^7.25.1",
     "eslint-plugin-react-hooks": "^4.2.0",
-    "expo-yarn-workspaces": "^1.5.1",
+    "expo-yarn-workspaces": "^1.7.0",
     "lerna": "^4.0.0",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.4.0",

